Tools/PostProcessing/plot_distribution_mapping/data/LBC_3D.txt

```diff
@@ -0,0 +1,5 @@
365]lev_box_45() [366]i_low_box_45() [367]j_low_box_45() [368]k_low_box_45() [369]gpu_ID_box_45() [370]hostname_box_45() [371]cost_box_46() [372]proc_box_46() [373]lev_box_46() [374]i_low_box_46() [375]j_low_box_46() [376]k_low_box_46() [377]gpu_ID_box_46() [378]hostname_box_46() [379]cost_box_47() [380]proc_box_47() [381]lev_box_47() [382]i_low_box_47() [383]j_low_box_47() [384]k_low_box_47() [385]gpu_ID_box_47() [386]hostname_box_47() [387]cost_box_48() [388]proc_box_48() [389]lev_box_48() [390]i_low_box_48() [391]j_low_box_48() [392]k_low_box_48() [393]gpu_ID_box_48() [394]hostname_box_48() [395]cost_box_49() [396]proc_box_49() [397]lev_box_49() [398]i_low_box_49() [399]j_low_box_49() [400]k_low_box_49() [401]gpu_ID_box_49() [402]hostname_box_49() [403]cost_box_50() [404]proc_box_50() [405]lev_box_50() [406]i_low_box_50() [407]j_low_box_50() [408]k_low_box_50() [409]gpu_ID_box_50() [410]hostname_box_50() [411]cost_box_51() [412]proc_box_51() [413]lev_box_51() [414]i_low_box_51() [415]j_low_box_51() [416]k_low_box_51() [417]gpu_ID_box_51() [418]hostname_box_51() [419]cost_box_52() [420]proc_box_52() [421]lev_box_52() [422]i_low_box_52() [423]j_low_box_52() [424]k_low_box_52() [425]gpu_ID_box_52() [426]hostname_box_52() [427]cost_box_53() [428]proc_box_53() [429]lev_box_53() [430]i_low_box_53() [431]j_low_box_53() [432]k_low_box_53() [433]gpu_ID_box_53() [434]hostname_box_53() [435]cost_box_54() [436]proc_box_54() [437]lev_box_54() [438]i_low_box_54() [439]j_low_box_54() [440]k_low_box_54() [441]gpu_ID_box_54() [442]hostname_box_54() [443]cost_box_55() [444]proc_box_55() [445]lev_box_55() [446]i_low_box_55() [447]j_low_box_55() [448]k_low_box_55() [449]gpu_ID_box_55() [450]hostname_box_55() [451]cost_box_56() [452]proc_box_56() [453]lev_box_56() [454]i_low_box_56() [455]j_low_box_56() [456]k_low_box_56() [457]gpu_ID_box_56() [458]hostname_box_56() [459]cost_box_57() [460]proc_box_57() [461]lev_box_57() [462]i_low_box_57() [463]j_low_box_57() [464]k_low_box_57() [465]gpu_ID_box_57() [466]hostname_box_57() [467]cost_box_58() [468]proc_box_58() [469]lev_box_58() [470]i_low_box_58() [471]j_low_box_58() [472]k_low_box_58() [473]gpu_ID_box_58() [474]hostname_box_58() [475]cost_box_59() [476]proc_box_59() [477]lev_box_59() [478]i_low_box_59() [479]j_low_box_59() [480]k_low_box_59() [481]gpu_ID_box_59() [482]hostname_box_59() [483]cost_box_60() [484]proc_box_60() [485]lev_box_60() [486]i_low_box_60() [487]j_low_box_60() [488]k_low_box_60() [489]gpu_ID_box_60() [490]hostname_box_60() [491]cost_box_61() [492]proc_box_61() [493]lev_box_61() [494]i_low_box_61() [495]j_low_box_61() [496]k_low_box_61() [497]gpu_ID_box_61() [498]hostname_box_61() [499]cost_box_62() [500]proc_box_62() [501]lev_box_62() [502]i_low_box_62() [503]j_low_box_62() [504]k_low_box_62() [505]gpu_ID_box_62() [506]hostname_box_62() [507]cost_box_63() [508]proc_box_63() [509]lev_box_63() [510]i_low_box_63() [511]j_low_box_63() [512]k_low_box_63() [513]gpu_ID_box_63() [514]hostname_box_63()
```

Comment:
Should we check in those data files? We probably don't need them, do we? Instead let us document this maybe?

Reply:
I guess they aren't super-important, but they do allow someone to use the plotting scripts out-of-the box without generating data themselves.  if we want to reduce bloat I can remove the samples files and tutorial notebook (which does not make sense to me without ready-to-use data).  pictures could still be included in the documentation.

Reply:
@ax3l any opinion on the above?

Reply:
I think let's remove those data files and add a small section in the docs to describe the workflow on how to create the plots and data for it :) is that ok @mrowan137? :)

Reply:
sounds good!